wallets/handler.go

```diff
@@ -395,20 +360,20 @@ func wrapRequestCommandIntoInputData(data *commandspb.InputData, req *walletpb.S
 	}
 }
 
-func (h *Handler) getLoggedWallet(name string) (Wallet, error) {
+func (h *Handler) getLoggedWallet(name string) (wallet.Wallet, error) {
 	exists := h.store.WalletExists(name)
 	if !exists {
-		return nil, ErrWalletDoesNotExists
+		return nil, wallet.ErrWalletDoesNotExists
```

Comment:
shouldn't this error be defined in this package?

Reply:
You are right